bower.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "requirejs-text",
```

Comment:
Why not just "text"? I would think that for module ID references, it would be useful to have this installed as "text", or is the repo name more important?